packages/api/src/external/hie/__tests__/set-doc-query-progress.test.ts

```diff
@@ -227,6 +200,69 @@ describe("aggregateAndSetHIEProgresses", () => {
     });
   });
 
+  it("has CW with total less than success and errors", async () => {
+    const progress = createProgress({
+      manuelProg: { total: 2, successful: 10, errors: 4, status: "processing" },
+    });
+
+    const externalData: PatientExternalData = {
+      COMMONWELL: {
+        documentQueryProgress: {
+          download: progress,
+        },
+      },
+    };
+
+    const overallDocQueryProgress: DocumentQueryProgress = {
+      ...docQueryProgress,
+      convert: undefined,
+    };
+
+    const aggregateAndSetHIEProgressesResult = aggregateAndSetHIEProgresses(
+      overallDocQueryProgress,
+      externalData
+    );
+
+    expect(aggregateAndSetHIEProgressesResult).toEqual({
+      download: progress,
+      requestId,
+    });
+  });
+
+  it("has CW with total less than success and errors - CQ is correct", async () => {
```

Comment:
nit: when we name test functions, if we create names that are just like other ones but with an additional suffix, we make it so that we can't run the one with shorter name, as Jest locates tests by name and the one with suffix will also be executed because it matches the short name too. I.e., if we try to run: 
- "has CW with total less than success and errors"

...Jest will execute:
- "has CW with total less than success and errors"
- "has CW with total less than success and errors - CQ is correct"

What I do is try to avoid name clashing, like:
- "has correct CQ and CW with total less than success and errors - "

Since we're on test naming, I try to name them like this:
- `it("<expected-result> when <state>")`
- I find it helps to know what to expect when I read the test, both input and output.